ksql-parser/src/main/java/io/confluent/ksql/parser/properties/with/WithPropertiesDef.java

```diff
@@ -31,15 +31,15 @@
 import org.apache.kafka.common.config.AbstractConfig;
 
 /**
- * Base class for handling 'with clause' properties.
+ * Helper for handling 'with clause' properties.
  */
 @Immutable
-abstract class WithClauseProperties extends AbstractConfig {
+final class WithPropertiesDef extends AbstractConfig {
```

Comment:
I think we should call this PropertyHelper (or similar) as it doesn't seem specific in any way to with clause properties.

Reply:
Yep, happy with that.

Reply:
I'm not too fussed, as long as it doesn't contain "with" in the name.